### ISSUE TITLE
Can't run any lastversion commands on Ubuntu 18.04

### ISSUE THREAD

**Issue by cobalt2727** [dvershinin/lastversion#100]
```python
ubuntu@oracle-vm:/tmp$ lastversion webcord
Traceback (most recent call last):
  File "/home/ubuntu/.local/bin/lastversion", line 8, in <module>
    sys.exit(main())
  File "/home/ubuntu/.local/lib/python3.8/site-packages/lastversion/lastversion.py", line 557, in main
    res = latest(args.repo, args.format, args.pre, args.filter,
  File "/home/ubuntu/.local/lib/python3.8/site-packages/lastversion/lastversion.py", line 167, in latest
    release = project.get_latest(pre_ok=pre_ok, major=major)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/lastversion/GitHubRepoSession.py", line 525, in get_latest
    feed_entries = self.get_releases_feed_entries()
  File "/home/ubuntu/.local/lib/python3.8/site-packages/lastversion/GitHubRepoSession.py", line 495, in get_releases_feed_entries
    feed_contents = self.get_releases_feed_contents()
  File "/home/ubuntu/.local/lib/python3.8/site-packages/lastversion/GitHubRepoSession.py", line 473, in get_releases_feed_contents
    r = self.get('https://{}/{}/releases.atom'.format(self.hostname, self.repo), headers=headers)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/lastversion/GitHubRepoSession.py", line 189, in get
    r = super(GitHubRepoSession, self).get(url, **kwargs)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/requests/sessions.py", line 600, in get
    return self.request("GET", url, **kwargs)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/cachecontrol/adapter.py", line 48, in send
    cached_response = self.controller.cached_request(request)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/cachecontrol/controller.py", line 155, in cached_request
    resp = self.serializer.loads(request, cache_data, body_file)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/cachecontrol/serialize.py", line 95, in loads
    return getattr(self, "_loads_v{}".format(ver))(request, data, body_file)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/cachecontrol/serialize.py", line 190, in _loads_v4
    return self.prepare_response(request, cached, body_file)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/cachecontrol/serialize.py", line 141, in prepare_response
    return HTTPResponse(body=body, preload_content=False, **cached["response"])
TypeError: __init__() got an unexpected keyword argument 'strict'
```

This seems to be related to https://stackoverflow.com/questions/76175361/firebase-authentication-httpresponse-object-has-no-attribute-strict-status

**Comment by dvershinin** [comment]
Fixed in latest v2.4.15 by pinning urllib among other things.

**Comment by cobalt2727** [comment]
Thanks for the quick fix again!
